drivers/dextrade.js

```diff
@@ -0,0 +1,46 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { throttleMap, parseToFloat } = require('../lib/utils');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class DexTrade extends Driver {
+  /**
+   * @param {boolean} isMocked Set to true when stored tickers are used
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers(isMocked) {
+    const { data: symbols } = await request('https://api.dex-trade.com/v1/public/symbols');
+    const pairs = {};
+
+    symbols.forEach((el) => {
+      pairs[el.pair] = {
+        base: el.base,
+        quote: el.quote,
+      };
+    });
+
+    const tickers = throttleMap(Object.keys(pairs), async (pair) => {
+      const { data: ticker } = await request(`https://api.dex-trade.com/v1/public/ticker?pair=${pair}`);
+      const { base, quote } = pairs[pair];
+
+      return new Ticker({
+        base,
+        quote,
+        high: parseToFloat(ticker.high),
+        low: parseToFloat(ticker.low),
+        close: parseToFloat(ticker.last),
+        open: parseToFloat(ticker.open),
+        quoteVolume: parseToFloat(ticker.volume_24H),
```

Comment:
If I compare the ticker (https://api.dex-trade.com/v1/public/ticker?pair=BTCUSDT) with their interface (https://dex-trade.com/spot/trading/BTCUSDT) I see that they say that is has 80k volume and in the endpoint it is around 7. So this should be the baseVolume
```suggestion
        baseVolume: parseToFloat(ticker.volume_24H),
```

Reply:
@nickpater when I try to make a fixture again from this driver I think that their server breaks in some point of time when there is too much requests on this url `https://api.dex-trade.com/v1/public/ticker?pair=`. I constantly have `CloudflareError: 521, Web server is down`. Is it working maybe for you?